test.ts

```diff
@@ -0,0 +1,30 @@
+import * as test from 'tape';
+import { stub } from 'sinon';
+
+import * as request from 'supertest';
+import app from './src/index';
+import * as aws from './src/services/aws';
+
+const testSvg = '<svg viewBox="0 0 10 10"><circle cx="5" cy="5" r="2"/></svg>';
+
+test('svgthumb-api POST /', (t: test.Test): void => {
+  t.plan(3);
+
+  const consoleStub = stub(console, 'log');
+  const awsUploadStub = stub(aws, 'upload').returns(Promise.resolve('S3 URL'));
+  const server = app.listen();
+
+  request(server)
+    .post('/')
+    .set('Content-Type', 'text/plain')
+    .send(testSvg)
+    .expect('X-Powered-By', /svgthumb-api/)
+    .expect('S3 URL')
+    .end((error: Error): void => {
+      consoleStub.restore();
```

Comment:
I think we need to restore `awsUploadStub` too (or maybe preferable: use a [sandbox](http://sinonjs.org/releases/v1.17.7/sandbox/) and restore the whole set at once)

Reply:
Using `sinon@6.1.3` now, so apparently:

> Since sinon@5.0.0, the sinon object is a default sandbox. Unless you have a very advanced setup or need a special configuration, you probably want to just use that one.

Cool!